ci/src/dependencies/scanner/manager/bazel_rust_dependency_manager.py

```diff
@@ -110,8 +110,11 @@ def __transitive_bazel_string_to_dependency(bazel_string: str) -> typing.Optiona
             # ['zstd', 'sys', '2.0.2', 'zstd.1.5.2']
             for split_string in parse_result:
                 # 2.0.2
-                if isinstance(version.parse(split_string), version.Version):
-                    version_str = split_string
+                try:
```

Comment:
For context: https://github.com/dfinity/ic/pull/1946#discussion_r1794722416

Reply:
Done, @mbjorkqvist would you like to take another look as this touched dependencies there, too?